doc/manuals/admin/yum.md

```diff
@@ -16,32 +16,31 @@ Use this configuration for release repository:
 ```
 [fiware-release]
 name=FIWARE release repository
-baseurl=https://nexus.lab.fiware.org/repository/el/$releasever/$basearch/release
+baseurl=https://nexus.lab.fiware.org/repository/el/7/x86_64/release
```

Comment:
Can you elaborate a bit? Has been $releaserver and $basearch variables deprecated in recent RHEL/CentOS versions?

Reply:
releasever returns a different values in rh and centos (7Server and 7), so it's better to make it static. basearch alone will require explanation, so it's better to make it static too.
parameters already changed in [release.repo](https://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-release.repo) and  [nightly.repo](https://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-nightly.repo)

Reply:
Thanks for the explanation!

NTC